### PR TITLE
Update guides.md

### PR DESCRIPTION
**Pull request by thomas-zimmerman**
...with links to new tech-docs links:
* oryp10
* pany12
* launch_2
* launch_lite_1
* launch_heavy_1

### CODE REVIEW

File: content/guides.md [hunk 1]
```diff
@@ -26,14 +26,14 @@ To see ports, keyboard layouts, function keys, and product quickstart guides, fi
...
-| Galago Pro [(galp4)](https://tech-docs.system76.com/models/galp4/README.html) | Thelio Massive B1 [(thelio-massive-b1)](https://tech-docs.system76.com/models/thelio-massive-b1.2/README.html) |
-| Galago Pro [(galp5)](https://tech-docs.system76.com/models/galp5/README.html) |
-| Galago Pro [(galp6)](https://tech-docs.system76.com/models/galp6/README.html) |
+| Adder WS [(addw2)](https://tech-docs.system76.com/models/addw2/README.html) | Meerkat [(meer6)](https://tech-docs.system76.com/models/meer6/README.html)  | Luanch [(launch_2)](https://tech-docs.system76.com/models/launch_2/README.html) |
+| Bonobo WS [(bonw14)](https://tech-docs.system76.com/models/bonw14/README.html) | Thelio B4 [(thelio-b4)](https://tech-docs.system76.com/models/thelio-b4/README.html) | Launch Lite [(launch_lite_1)](https://tech-docs.system76.com/models/launch_lite_1/README.html) |
+| Darter Pro [(darp6)](https://tech-docs.system76.com/models/darp6/README.html) | Thelio Major [(thelio-major)](https://tech-docs.system76.com/models/thelio-major-intel-and-amd/README.html) |  Launch Heavy [(launch_heavy_1)](https://tech-docs.system76.com/models/launch_heavy_1/README.html) |
```
Comment (n3m0-22) [inline content/guides.md:31]: The link for `Thelio Major [(thelio-major)]` should be changed to `Thelio Major [(thelio-major-r3)]`

**Review by n3m0-22 — CHANGES_REQUESTED** [review]

File: content/guides.md [hunk 2]
```diff
@@ -28,13 +28,13 @@ To see ports, keyboard layouts, function keys, and product quickstart guides, fi
...
-| Galago Pro [(galp4)](https://tech-docs.system76.com/models/galp4/README.html) | Thelio Mira R1 [(thelio-mira-r1)](https://tech-docs.system76.com/models/thelio-mira-r1.0/README.html) |
-| Galago Pro [(galp5)](https://tech-docs.system76.com/models/galp5/README.html) | Thelio Mira B1 [(thelio-mira-b1)](https://tech-docs.system76.com/models/thelio-mira-b1.0/README.html) |
-| Galago Pro [(galp6)](https://tech-docs.system76.com/models/galp6/README.html) | Thelio Mira R3 [(thelio-mira-r3)](https://tech-docs.system76.com/models/thelio-mira-r3/README.html) |
-| Gazelle [(gaze15)](https://tech-docs.system76.com/models/gaze15/README.html) |
+| Darter Pro [(darp6)](https://tech-docs.system76.com/models/darp6/README.html) | Thelio Major [(thelio-major-b1-b2/r1-r2)](https://tech-docs.system76.com/models/thelio-major-b1-b2-r1-r2/README.html) |  Launch Heavy [(launch_heavy_1)](https://tech-docs.system76.com/models/launch_heavy_1/README.html) |
+| Darter Pro [(darp7)](https://tech-docs.system76.com/models/darp7/README.html) | Thelio Major [(thelio-major-r2)](https://tech-docs.system76.com/models/thelio-major-r3/README.html) |
```
Comment (n3m0-22) [inline content/guides.md:32]: The Thelio Major link text shows `thelio-major-r2` but links to `thelio-major-r3`

**Review by n3m0-22 — CHANGES_REQUESTED** [review]

**Review by ahoneybun — DISMISSED** [review]

**Review by n3m0-22 — APPROVED** [review]
Looks good

**Comment by ahoneybun** [comment]
I'm going to add the new serw13 and addw3 tech docs then we can review this again.

**Review by ahoneybun — APPROVED** [review]

**Review by ahoneybun — APPROVED** [review]

**Review by n3m0-22 — APPROVED** [review]
All text and links are correct now. Looks good.

**Pull request merged**